packages/cva/src/index.ts

```diff
@@ -254,3 +292,31 @@ export const defineConfig: DefineConfig = (options) => {
 };
 
 export const { compose, cva, cx } = defineConfig();
+
+export interface CreateSchema {
+  <_ extends InternalOnly, Component, Variants>(
+    component: Component &
+      (Component extends ReturnType<CVA>
+        ? { _cva: CVAConfig<Variants> }
+        : never),
+  ): {
+    [Variant in keyof Variants]: ReadonlyArray<
+      StringToBoolean<keyof Variants[Variant]>
+    >;
+  };
+}
+
+export const getSchema: CreateSchema = (component) => {
```

Comment:
Wonder if it's also worth returning the type (`string` or `boolean`) info given we return a boolean string rather than the boolean, which is a little confusing